docs/docsite/rst/getting_started_ee/setup_environment.rst

```diff
@@ -0,0 +1,37 @@
+.. _setting_up_environment:
+
+###########################
+Setting up your environment
+###########################
+
+Complete the following steps to set up a local environment for your first execution environment:
+
+1. Ensure the following packages are installed on your system:
+
+* ``podman`` or ``docker``
+* ``python3``
+
+If you use the DNF package manager, install them as follows:
+
+.. code-block:: bash
+
+  sudo dnf install -y podman python3
+
+2. Upgrade the `pip` package manager and install `ansible-navigator` to run EEs.
+
+Installing `ansible-navigator` lets you run EEs.
+It includes the `ansible-builder` package to build EEs.
+
+.. code-block:: bash
+
+  python3 -m pip install --upgrade pip ansible-navigator
+
```

Comment:
I'm inclined to say don't add upgrade pip. it's not required, and there may be valid reasons someone doesn't want to impact their version of pip.

Reply:
I agree, my original suggestion also was just `pip install`, let's change this back.